conda/recipes/pyraft/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+ucx_version:
+  - "1.12.1"
```

Comment:
I think this should be `>=1.12.1`

Reply:
The intent here is to ensure we have a fixed version at build time so that this sets a minimum version at runtime.

Reply:
Oh, I thought we wanted to build using the latest version available. What's the difference between doing this and using `pin_compatible` for the runtime dependency?

Reply:
We are using the latest version since 1.12.1 is latest. However we may want more flexibility if a later 1.12.x breaks something and we need to pin an older version in the metapackage. Also we may want to control which version we build against to make it easier to align with other packages that have `ucx` constraints as well.

FWIW am proposing something similar in conda-forge ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/2847 ) though maybe we can wait to see what others think about that and update if needed.